Sources/SwiftFormat/PrettyPrint/PrettyPrint.swift

```diff
@@ -449,7 +449,11 @@ public class PrettyPrinter {
 
     // Print out the number of spaces according to the size, and adjust spaceRemaining.
     case .space(let size, _):
-      outputBuffer.enqueueSpaces(size)
+      if configuration.allowWhitespaceOnlyLines, outputBuffer.isAtStartOfLine {
+        outputBuffer.write(String())
```

Comment:
Can you leave a brief comment here explaining why we do this? To a random observer, asking it to write an empty string feels odd, but it's because we need the buffer to still do its other logic (flushing the pending indentation, etc.).

I'd also just write it as `outputBuffer.write("")`; same effect but easier to read/scan.

Reply:
I've added a comment and made the modification!